<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>523</width>
    <height>508</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLineEdit" name="server_addr">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="server_port">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>设备服务器地址:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>设备服务器端口号:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Connect_server">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>连接服务器</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>60</y>
      <width>291</width>
      <height>381</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>40</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>服务器回复消息:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SEND_SSID_AND_PASSWORD">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>测试发送ssid和密码</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SEND_SERVER_IP_AND_PORT">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>400</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>测试发送ip和端口</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="input_ap">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="input_server_ip">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>130</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>wifi_ap</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>180</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>wifi_password</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="input_ap_password">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>290</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>server_ip</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>340</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>server_port</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="input_server_port">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>360</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="Disconnect_server">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>90</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>服务器断开</string>
    </property>
   </widget>
   <zorder>server_addr</zorder>
   <zorder>server_port</zorder>
   <zorder>label_5</zorder>
   <zorder>label_6</zorder>
   <zorder>Connect_server</zorder>
   <zorder>textBrowser</zorder>
   <zorder>label_7</zorder>
   <zorder>SEND_SERVER_IP_AND_PORT</zorder>
   <zorder>SEND_SSID_AND_PASSWORD</zorder>
   <zorder>input_ap</zorder>
   <zorder>input_server_ip</zorder>
   <zorder>label_8</zorder>
   <zorder>label_9</zorder>
   <zorder>input_ap_password</zorder>
   <zorder>label_10</zorder>
   <zorder>label_11</zorder>
   <zorder>input_server_port</zorder>
   <zorder>Disconnect_server</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>523</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
